tools/stress-cluster/cluster/kubernetes/stress-infrastructure/templates/stresswatcher.yaml

```diff
@@ -1,36 +1,36 @@
-{{ $ctx := deepCopy . | merge (dict "Scenario" "watcher" ) -}}
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: stress-watcher
-  namespace: {{ .Release.Namespace }}
-  labels:
-    app: stress-watcher
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: stress-watcher
-  template:
-    metadata:
-      namespace: {{ .Release.Namespace }}
-      labels:
-        app: stress-watcher
-    spec:
-      nodeSelector:
-        sku: 'default'
-      initContainers:
-        # Init container template for injecting secrets
-        # (e.g. app insights instrumentation key, azure client credentials)
-        {{- include "stress-test-addons.init-env" $ctx | nindent 8 }}
-      containers:
-      - name: stresswatcher
-        imagePullPolicy: Always
-        image: azsdkengsys.azurecr.io/stress/watcher:{{ .Values.tag }}
-        command: ["dotnet", "Stress.Watcher.dll"]
-        {{- include "stress-test-addons.container-env" $ctx | nindent 8 }}
-      volumes:
-        # Volume template for mounting secrets
-        {{- include "stress-test-addons.env-volumes" $ctx | nindent 8 }}
-        # Volume template for mounting azure file share for debugging
-        {{- include "stress-test-addons.debug-file-volumes" $ctx | nindent 8 }}
+{{ $ctx := fromYaml (include "stress-test-addons.util.mergeStressContext" (list . "watcher" )) }}

```

Comment:
Given the full diff here, I'm suspecting either bad line endings were added, or they were finally removed?

Reply:
The LF/CRLF settings on my VSCode is a bit weird today, thanks for catching this